incubator/kiam/templates/server-daemonset.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    app: {{ template "kiam.name" . }}
+    chart: {{ template "kiam.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: {{ template "kiam.fullname" . }}-server
+  namespace: {{ .Release.Namespace }}
+spec:
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: {{ quote .Values.server.prometheus.scrape }}
+        prometheus.io/port: {{ quote .Values.server.prometheus.port }}
```

Comment:
Remove hard-coded annotations. Users that need them can configure them.

Reply:
Fixed.